package.json

```diff
@@ -1,13 +1,17 @@
 {
   "name": "nhsuk-frontend",
-  "version": "0.5.0",
+  "version": "0.1.6",
```

Comment:
Make sure you get latest version from master, this shouldn't be changed to an earlier version.